code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -59,3 +59,7 @@
 	var/next_hallucination = 0
 	var/cpr_time = 1 //CPR cooldown.
 	var/damageoverlaytemp = 0
+
+	var/pain_shock_stage = 0
+	var/pain_level = 0 //Gets overwritten every tick! If you want to add pain "resistance" or constain pain, see update_pain_level()
+	var/pain_numb = 0 // When this is set, the mob isn't affected by shock or pain.
```

Comment:
please make this a trait

Reply:
that's not a bool, that's like stuttering/slurring/eye_blurry/etc,  it gets decreased every tick

Reply:
ideally you want a status effect for that then, but a trait that does this is generally easier and more appropriate when using chems (see pax for example)

Reply:
done